usagov/vote-gov-drupal
# Feature/VOTE-449 add Hmong translation

## Jira ticket

[VOTE-449](https://cm-jira.usa.gov/browse/VOTE-449)

## Description

Add PO and Config files to enable and provide string translations for Hmong.

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

1. Locally, run `lando retune -y` to import config files and string translations from PO file.

### QA/Test

1.  Visit http://vote-gov.lndo.site/admin/config/regional/language and confirm that Hmong language is listed.
2. Visit http://vote-gov.lndo.site/admin/config/regional/translate
3. Do a search using the following: Translation: Hmong, Search In: Only translated strings
4. Confirm that the search returns a list of translated strings for Hmong.

## Checklist for the Developer




## Checklist for the Peer Reviewers
